projects/lz4/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/lz4/lz4"
+primary_contact: "Yann.collet.73@gmail.com"
+auto_ccs:
+  - "NickRTerrell@gmail.com"
+  - "cyan@fb.com"
+  - "terrelln@fb.com"
+sanitizers:
```

Comment:
Please add the `memory` sanitizer. Lz4 should be MSAN clean.

Reply:
Fixed.